blas/gonum/level2cmplx128.go

```diff
@@ -209,6 +209,130 @@ func (Implementation) Zgeru(m, n int, alpha complex128, x []complex128, incX int
 	}
 }
 
+// Zhemv performs the matrix-vector operation
+//  y = alpha * A * x + beta * y
+// where alpha and beta are scalars, x and y are vectors, and A is an n×n
+// Hermitian matrix. The imaginary parts of the diagonal elements of A are
+// ignored and assumed to be zero.
+func (Implementation) Zhemv(uplo blas.Uplo, n int, alpha complex128, a []complex128, lda int, x []complex128, incX int, beta complex128, y []complex128, incY int) {
+	if uplo != blas.Upper && uplo != blas.Lower {
+		panic(badUplo)
+	}
+	checkZMatrix('A', n, n, a, lda)
+	checkZVector('x', n, x, incX)
+	checkZVector('y', n, y, incY)
+
+	if n == 0 || (alpha == 0 && beta == 1) {
+		return
+	}
+
+	var kx int
+	if incX < 0 {
+		kx = (1 - n) * incX
+	}
+	var ky int
+	if incY < 0 {
+		ky = (1 - n) * incY
+	}
+
```

Comment:
The comments look quite useful.

Reply:
Ok, I added them. Could you please re-approve?